packages/block-editor/src/components/inspector-controls-tabs/styles-tab.js

```diff
@@ -32,6 +32,7 @@ const StylesTab = ( { blockName, clientId, hasBlockStyles } ) => {
 				label={ __( 'Color' ) }
 				className="color-block-support-panel__inner-wrapper"
 			/>
+			<InspectorControls.Slot group="filter" />
```

Comment:
@jasmussen is this a proper order for the filters? I put it after color because it's what made sense to me